Velaptor/Input/KeyboardState.cs

```diff
@@ -326,30 +326,25 @@ public char KeyToChar(KeyCode key)
                     return WithShiftSymbolCharacters[key];
                 }
 
-                if (SymbolKeys.Contains(key))
-                {
-                    return WithShiftSymbolCharacters[key];
-                }
+                return WithShiftSymbolCharacters[key];
             }
-            else
+
+            if (LetterKeys.Contains(key))
             {
-                if (LetterKeys.Contains(key))
-                {
 #pragma warning disable CA1304 // Specify CultureInfo
-                    return key == KeyCode.Space ? ' ' : key.ToString().ToLower()[0];
+                return key == KeyCode.Space ? ' ' : key.ToString().ToLower()[0];
 #pragma warning restore CA1304 // Specify CultureInfo
-                }
+            }
 
-                if (SymbolKeys.Contains(key))
-                {
-                    return NoShiftSymbolCharacters[key];
-                }
+            if (SymbolKeys.Contains(key))
+            {
+                return NoShiftSymbolCharacters[key];
+            }
 
-                // When the shift is is up, the standard number keys return numbers.
-                if (StandardNumberKeys.Contains(key))
-                {
-                    return NoShiftStandardNumberCharacters[key];
-                }
+            // When the shift is is up, the standard number keys return numbers.
```

Comment:
Refactor comment as below:

```suggestion
            // When the shift is up, the standard number keys return numbers.
```